prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml

```diff
@@ -6,14 +6,11 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+      - image: quay.io/ansible/ansible-runner:latest
```

Comment:
Given that, according to the README, we support Ansible 2.9 or higher, can we tests against this ansible version?  That way we can help ensure we keep this support.

Reply:
Modifying it to use 2.9 stable version.
Yes, the testing did work for me successfully from inside a GKE pod:
```
jcnarasimhan@jon2:~/DriveFS/My Drive/bmaas/test_automation/oss_prow_onboarding/running_cleanup_install_oracle$ kubectl exec -it test-pods-namespace-lo-podname -n test-pods -- /bin/bash
bash-4.4# cd /root/bms-toolkit/
bash-4.4# ls -l sydney-1.out.thegabba.brutecleanup 
-rw-r--r-- 1 root root 24974 Nov 16 02:03 sydney-1.out.thegabba.brutecleanup
bash-4.4# tail sydney-1.out.thegabba.brutecleanup 

TASK [brute-ora-cleanup : Server reboot] ***************************************
skipping: [linuxserver44.orcl]

TASK [brute-ora-cleanup : Re-mount swlib] **************************************
skipping: [linuxserver44.orcl]

PLAY RECAP *********************************************************************
linuxserver44.orcl         : ok=21   changed=8    unreachable=0    failed=0    skipped=14   rescued=0    ignored=5   
```